### PR TITLE
Enable collection of notifications

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: heka/meta/heka.yml [hunk 1]
```diff
@@ -159,13 +187,18 @@ remote_collector:
...
-{%- if remote_collector.influxdb_host is defined %}
   encoder:
+{%- if remote_collector.influxdb_host is defined %}
     influxdb:
       engine: payload
       append_newlines: false
       prefix_ts: false
 {%- endif %}
+{%- if log_collector.elasticsearch_host is defined %}
+  encoder:
```
Comment (SwannCroiset) [inline heka/meta/heka.yml]: duplicate "encoder" key L198

**Review by SwannCroiset — CHANGES_REQUESTED** [review]

File: heka/meta/heka.yml (hunk 1 above)
Comment (simonpasquier) [reply heka/meta/heka.yml]: done

**Comment by simonpasquier** [comment]
@SwannCroiset can you review this again?

**Review by elemoine — APPROVED** [review]

File: heka/meta/heka.yml [hunk 2]
```diff
@@ -139,13 +139,41 @@ remote_collector:
...
+{%- endif %}
   input:
     heka_collectd:
       engine: http
       address: 127.0.0.1
       port: 8326
       decoder: collectd_decoder
       splitter: NullSplitter
+{%- if remote_collector.amqp_host is defined %}
+{%- for notification_level in ['info', 'warn', 'error'] %}
```
Comment (elemoine) [inline heka/meta/heka.yml]: Maybe a bit personal, but I'd use a tuple rather than a list here.

Comment (simonpasquier) [reply heka/meta/heka.yml]: done

**Review by SwannCroiset — APPROVED** [review]

**Pull request merged**